Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -592,28 +592,29 @@ - (void)showCapture {
  */
 - (void)addContainerEmptyView
 {
-    if (self.containerEmptyView != nil && self.containerEmptyView.superview != nil) {
-        [self selectEmptyView];
+    if (self.emptyViewContainer != nil && self.emptyViewContainer.superview != nil) {
+        self.emptyViewContainer.hidden = false;
+        [self addEmptyViewToContainer];
```

Comment:
I think we can now get rid of these checks since we'll be sure we'll only call this once. Same for the checks in `addEmptyViewControllerToView` and `addEmptyViewToView`. What do you think?

Reply:
I agree to remove for `emptyViewContainer`, it works fine, but for the `addEmptyViewControllerToView` and `addEmptyViewToView` we need it, if you consider my previous comment about the `viewDidLoad`, with this, we avoid adding two views on the container.